### PR TITLE
Add non-status Toast

### PR DESCRIPTION
**Pull request by ThusharaJ07**

### CODE REVIEW

File: packages/theme/css/characteristics/container.css [hunk 1]
```diff
@@ -5,6 +5,7 @@
   --salt-container-primary-background-disabled: var(--salt-palette-neutral-primary-background-disabled);
   --salt-container-primary-borderColor: var(--salt-palette-neutral-primary-border);
   --salt-container-primary-borderColor-disabled: var(--salt-palette-neutral-primary-border-disabled);
+  --salt-container-primary-borderColor-default: var(--salt-palette-neutral-primary-border-default);
```
Comment (joshwooding) [inline packages/theme/css/characteristics/container.css:8]:
```suggestion
```
These tokens aren't needed

File: packages/theme/css/palette/neutral.css [hunk 2]
```diff
@@ -17,13 +17,15 @@
   --salt-palette-neutral-backdrop: var(--salt-color-white-fade-backdrop);
   --salt-palette-neutral-secondary-separator: var(--salt-color-black-fade-separatorOpacity-secondary);
   --salt-palette-neutral-tertiary-separator: var(--salt-color-black-fade-separatorOpacity-tertiary);
+  --salt-palette-neutral-primary-border-default: var(--salt-color-gray-50);
```
Comment (joshwooding) [inline packages/theme/css/palette/neutral.css:20]:
```suggestion
```

File: packages/theme/css/palette/neutral.css [hunk 3]
```diff
@@ -17,13 +17,15 @@
...
 }
 
 .salt-theme[data-mode="dark"] {
   --salt-palette-neutral-primary-background: var(--salt-color-gray-800);
   --salt-palette-neutral-primary-background-disabled: var(--salt-color-gray-800-fade-background);
   --salt-palette-neutral-primary-background-readonly: var(--salt-color-gray-800-fade-background-readonly);
   --salt-palette-neutral-primary-border: var(--salt-color-gray-300);
+  --salt-palette-neutral-primary-border-default: var(--salt-color-gray-300);
```
Comment (joshwooding) [inline packages/theme/css/palette/neutral.css:28]:
```suggestion
```

File: packages/core/src/toast/Toast.css [hunk 4]
```diff
@@ -53,3 +54,8 @@
 .saltToast-success {
   --toast-borderColor: var(--salt-status-success-borderColor);
 }
+
+/* Styles applied when no status is passed*/
+.saltToast-default {
+  --toast-borderColor: var(--salt-container-primary-borderColor-default);
```
Comment (joshwooding) [inline packages/core/src/toast/Toast.css:60]:
```suggestion
  --toast-borderColor: var(--salt-container-primary-borderColor);
```

File: packages/core/src/toast/Toast.tsx [hunk 5]
```diff
@@ -30,12 +30,18 @@ export const Toast = forwardRef<HTMLDivElement, ToastProps>(function Toast(
 
   return (
     <div
-      className={clsx(withBaseName(), withBaseName(status), className)}
+      className={clsx(
+        withBaseName(),
+        withBaseName(status ?? "default"),
```
Comment (joshwooding) [inline packages/core/src/toast/Toast.tsx:35]: Can this be implemented in the same way as Tooltip please

**Review by joshwooding — CHANGES_REQUESTED** [review]
Left a few comments, the tokens differ slightly between Figma and code but no new tokens should be needed. Also we need to add the default toast variant to the QA story.

File: packages/theme/css/characteristics/container.css (hunk 1 above)
Comment (ThusharaJ07) [reply packages/theme/css/characteristics/container.css]: removed

File: packages/theme/css/palette/neutral.css (hunk 2 above)
Comment (ThusharaJ07) [reply packages/theme/css/palette/neutral.css]: removed

File: packages/theme/css/palette/neutral.css (hunk 3 above)
Comment (ThusharaJ07) [reply packages/theme/css/palette/neutral.css]: removed

File: packages/core/src/toast/Toast.css (hunk 4 above)
Comment (ThusharaJ07) [reply packages/core/src/toast/Toast.css]: modified

File: packages/core/src/toast/Toast.tsx (hunk 5 above)
Comment (ThusharaJ07) [reply packages/core/src/toast/Toast.tsx]: made it as per tooltip

**Review by origami-z — COMMENTED** [review]
✅  will need to add changeset

**Review by joshwooding — APPROVED** [review]

**Review by joshwooding — APPROVED** [review]

**Review by Fercas123 — APPROVED** [review]

**Pull request merged**